app/Commands/Host.php

```diff
@@ -128,6 +146,11 @@ private function defaultDomainNameFromKey($key){
         return $key;
     }
 
+    private function createProject()
+    {
+        $shellOutput = shell_exec("cd {$this->folder} && composer create-project {$this->composerProject} {$this->name}");
```

Comment:
Also using composer create-project vs laravel new avoids issues for those who don't have the laravel installer